src/cargo/ops/cargo_update.rs

```diff
@@ -14,6 +14,7 @@ use crate::util::toml_mut::manifest::LocalManifest;
 use crate::util::toml_mut::upgrade::upgrade_requirement;
 use crate::util::{style, OptVersionReq};
 use crate::util::{CargoResult, VersionExt};
+use anyhow::Context;
```

Comment:
nit: `Context` is a name so general that may conflict with other structs.

```suggestion
use anyhow::Context as _;
```